static/app/utils/issueTypeConfig/cronConfig.tsx

```diff
@@ -7,8 +7,8 @@ const cronConfig: IssueCategoryConfigMapping = {
     actions: {
       archiveUntilOccurrence: {enabled: true},
       delete: {
-        enabled: false,
-        disabledReason: t('Not yet supported for cron issues'),
+        enabled: true,
+        disabledReason: '',
```

Comment:
These `disabledReason` fields are optional, we can just omit them instead of using an empty string.